dept44-starter/src/main/java/se/sundsvall/dept44/configuration/LogbookConfiguration.java

```diff
@@ -30,24 +31,22 @@
 import org.zalando.logbook.autoconfigure.LogbookAutoConfiguration;
 import org.zalando.logbook.json.JsonHttpLogFormatter;
 
-import com.fasterxml.jackson.databind.ObjectMapper;
-
 @AutoConfigureBefore(LogbookAutoConfiguration.class)
 public class LogbookConfiguration {
 
 	private final String loggerName;
 	private final Set<String> excludedPaths;
-	private final Map<String, String> exclusionFiltersJsonPath;
+	private final Map<String, String> exclusionFiltersJsonProperty;
 	private final Map<String, String> exclusionFiltersXPath;
 
 	LogbookConfiguration(
-		@Value("#{${logbook.exclusionfilters.json-path: {} }}") Map<String, String> exclusionFiltersJsonPath,
+		@Value("#{${logbook.exclusionfilters.json-property: {} }}") Map<String, String> exclusionFiltersJsonProperty,
```

Comment:
Antar det här blir en "brytande" ändring?
Dom här tjänsterna måste vi in och ändra i innan vi uppar dept44-version för dom:
 - process-engine-parking-permit
 - api-service-support-management
 - api-service-digital-mail-sender
 - api-service-web-message-sender